src/Microsoft.AspNet.Mvc.TagHelpers/MatcherExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.Mvc;
+
+namespace Microsoft.Framework.FileSystemGlobbing
+{
+    /// <summary>
+    /// Extension methods for <see cref="Matcher"/>.
+    /// </summary>
+    public static class MatcherExtensions
+    {
+        /// <summary>
+        /// Adds include and exclude patterns.
+        /// </summary>
+        /// <param name="matcher">The <see cref="Matcher"/>.</param>
+        /// <param name="includePatterns">The set of include globbing patterns.</param>
+        /// <param name="excludePatterns">The set of exclude globbing patterns.</param>
+        public static void AddPatterns([NotNull]this Matcher matcher,
```

Comment:
Formatting: 

* Have the params stacked (so matcher is on the next line)
* Space after `[NotNull]`